share/errmsg-utf8.txt

```diff
@@ -5080,12 +5080,12 @@ ER_FEATURE_DISABLED
         spa "El recurso '%s' fue deshabilitado; usted necesita construir MySQL con '%s' para tener eso funcionando"
         swe "'%s' är inte aktiverad; För att aktivera detta måste du bygga om MySQL med '%s' definierad"
 ER_OPTION_PREVENTS_STATEMENT  
```

Comment:
The original code added a new ER_OPTION_PREVENTS_STATEMENT_EXTRA_INFO code. It would be better to keep to the original code instead. The value of the error code returned would match the existing 5.6 version. The return codes would be different between servers that have a master vs. those that do not, without having to parse the return string to determine if host:port were appended to it.

Reply:
This change matches existing 5.6 version:
`ER_OPTION_PREVENTS_STATEMENT_EXTRA_INFO` was removed with https://github.com/facebook/mysql-5.6/commit/9f18725 in favor of `ER_OPTION_PREVENTS_STATEMENT` with additional string param.

Reply:
I only changed output to match upstream when additional parameter is empty string e.g.
```
ERROR HY000: The MySQL server is running with the --secure-file-priv option so it cannot execute this statement
```
instead of
```
ERROR HY000: The MySQL server is running with the --secure-file-priv option so it cannot execute this statement. 
```
in 5.6 with additional `. ` at the end.

Reply:
Ah, thanks. I missed checking the last diff that reverted this behavior.